src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,60 @@
+package step2;
+
+public class Calculator {
```

Comment:
계산기 클래스를 Utilities 클래스로 만들어주셨네요. 하지만, 다른 곳에서 new Calculator()로 인스턴스를 생성하면 어떻게 될까요? 인스턴스를 생성할 필요가 없는 유틸 클래스는 생성자를 막아둘 필요가 있습니다. 

```suggestion
public class Calculator {
    private Calculator(){...}
```